src/app/demo/demo.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit, ViewContainerRef } from "@angular/core";
+import {
+  NgbModal,
+  ModalDismissReasons,
+  NgbModalRef
+} from "@ng-bootstrap/ng-bootstrap";
+import { constants } from "../constants";
+
+@Component({
+  selector: "app-demo",
+  templateUrl: "./demo.component.html",
+  styleUrls: ["./demo.component.scss"]
+})
+export class DemoComponent implements OnInit {
+  public isCardFreeze: boolean;
+  clicked = true;
```

Comment:
clicked=true is not used anywhere, please remove it. Also remove unused imports and properties, methods